docker/php/env

```diff
@@ -0,0 +1,6 @@
+SYMFONY__ENV__DATABASE_DRIVER="pdo_sqlite"
```

Comment:
Not sure. This file does not refer to a *sqlite* container, and could contain more `SYMFONY__` env variables or other kind of variables for *php* container.